module-domain/src/main/kotlin/site/yourevents/invitationinformation/port/out/InvitationInformationPersistencePort.kt

```diff
@@ -0,0 +1,7 @@
+package site.yourevents.invitationinformation.port.out
+
+import site.yourevents.invitationinformation.domain.InvitationInformation
+
+interface InvitationInformationPersistencePort {
+    fun saveInvitationInformation(invitationInformation: InvitationInformation): InvitationInformation
```

Comment:
해당 메서드는 간단하게 `save`로 수정해도 좋을 것 같아요!! 그렇게 변경해도 이해하는 데 큰 문제는 없어보입니다..!
```suggestion
    fun save(invitationInformation: InvitationInformation): InvitationInformation
```

Reply:
저도 물론 save로 할까 고민을 하였는데 저장 메서드가 guest, invitation, invitationInformation, invitationThumbnail 전부 존재하기에 이름을 구분해주면 조금 더 좋지 않을까라는 생각을 하였습니다. 이정도까지 하는건 투머치일까요? 어떻게 생각하시나요?

Reply:
어떤 것을 저장하느냐에 대한 부분은 현재 도메인 별로 인터페이스가 분리되어 있기 때문에, 메서드명을 간결하게 수정해도 괜찮을 것 같다는 생각입니다!

Reply:
넵 알겠습니다!! 정완님 방법대로 간결하게 수정하는 것이 좋다고 생각됩니다.  수정하였습니다!!